src/intl/en/page-layer-2.json

```
@@ -58,6 +58,7 @@
   "layer-2-dyor-3": "Go to L2BEAT",
   "layer-2-use-layer-2-title": "Use layer 2",
   "layer-2-use-layer-2-1": "Now that you understand why layer 2 exists and how it works, let's get you up and running!",
+  "layer-2-use-layer-2-note": "<strong>NOTE:</strong> When bridging over and using layer 2, it is important to note that you will control the address for your <a href=\"/glossary/#eoa\">EOA account</a> (an account where only a single private key controls the account) just like on Ethereum Mainnet. However, if you are using a <a href=\"/glossary/#contract-account\">contract account</a>, such as <a href=\"https://gnosis-safe.io/\"  target=”_blank”>Gnosis Safe</a> or <a href=\"https://www.argent.xyz/\"  target=”_blank”>Argent</a>, you will not have control over this address on a layer 2 until you redeploy your contract account to that address on the layer 2. If you are bridging or sending funds to a contract account, and you do not control this address for the contract account, your funds may be lost.",
```

We might move this content elsewhere later; maybe something like this would be a better key?

```suggestion
  "layer-2-contract-accounts": "<strong>NOTE:</strong> When bridging over and using layer 2, it is important to note that you will control the address for your <a href=\"/glossary/#eoa\">EOA account</a> (an account where only a single private key controls the account) just like on Ethereum Mainnet. However, if you are using a <a href=\"/glossary/#contract-account\">contract account</a>, such as <a href=\"https://gnosis-safe.io/\"  target=”_blank”>Gnosis Safe</a> or <a href=\"https://www.argent.xyz/\"  target=”_blank”>Argent</a>, you will not have control over this address on a layer 2 until you redeploy your contract account to that address on the layer 2. If you are bridging or sending funds to a contract account, and you do not control this address for the contract account, your funds may be lost.",
```